go/obscuronode/host/host.go

```diff
@@ -485,12 +498,21 @@
 	for _, tx := range b.Transactions() {
 		t := a.txHandler.UnPackTx(tx)
 		if t != nil && t.TxType == obscurocommon.RequestSecretTx {
-			txData := &obscurocommon.L1TxData{
-				TxType:      obscurocommon.StoreSecretTx,
-				Secret:      a.Enclave.FetchSecret(t.Attestation),
-				Attestation: t.Attestation,
+			att, err := nodecommon.DecodeAttestation(t.Attestation)
+			if err != nil {
```

Comment:
Should there be a `continue` here ?

Reply:
Yes, good spot - that would have been very annoying